include/inexor/vulkan-renderer/wrapper/framebuffer.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <vulkan/vulkan_core.h>
+
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace inexor::vulkan_renderer::wrapper {
+
+class Framebuffer {
+private:
+    VkDevice device;
```

Comment:
Default should be `nullptr`?

Reply:
Initializing Vulkan objects to `VK_NULL_HANDLE?  or `nullptr` it not really necessary if we create them in the constructor anyways.